src/Emulator/Cores/cores-riscv_NET.csproj

```diff
@@ -40,6 +40,7 @@
     <Compile Include="RiscV\RiscVInstructionPythonEngine.cs" />
     <Compile Include="RiscV\RiscVCsrPythonEngine.cs" />
     <Compile Include="RiscV\CV32E40P.cs" />
+    <Compile Include="RiscV\Andes_A45.cs" />
```

Comment:
I dont see the Andes_A45.cs file being added in this PR. Shouldnt that be added?

Reply:
Also I see a lot of other changes in the renode_rs for infrastructure, why arent they folded in here?

Reply:
> I dont see the Andes_A45.cs file being added in this PR. Shouldnt that be added?

Andes_A45.cs was previously added to master without a PR. It is already in /Emulator/Cores/RiscV

Reply:
> Also I see a lot of other changes in the renode_rs for infrastructure (8 in total), why arent they folded in here?

This is a specific branch for adding the ATCUART100 model. The exception is adding Andes_A45.cs to be compiled.

Are the changes you mention related to the instrumentation added to the code to track the execution and available in another branch?

Reply:
I see. How do we merge the changes, I mean if there is a commit to a **branch** of the renode-infrastructure_rs, do you point to that commit in the renode_rs and then when you are committing in renode_rs at that you make it point to the merged commit in the renode-infrastructure_rs?
For example, when we merge this PR, it will create a new commit on main branch of this repository. Then we need to make the renode_rs point to that new commit in the main of this repository in the new PR...right?